dev/NavigationView/NavigationView.cpp

```diff
@@ -1399,6 +1400,32 @@ void NavigationView::OnItemsContainerSizeChanged(const winrt::IInspectable& send
     UpdatePaneLayout();
 }
 
+void NavigationView::UpdateOpenPaneWidth(double width)
+{
+    // In top nav, there is no adaptive pane layout
+    if (IsTopNavigationView())
+    {
+        return;
+    }
+
+    if (!m_rootSplitView)
```

Comment:
can combine with topnav check above

Reply:
Cpp guidelines suggest avoiding early returns. In this case I don't there is a real need for it.